### ISSUE TITLE
Can't build zimg-9999

### ISSUE THREAD

**Issue by Kagami** [4re/vapoursynth-portage#32]
Hi. I get this error when trying to build zimg from git:

```
$ emerge zimg
...
x86_64-pc-linux-gnu-g++ -DPACKAGE_NAME=\"zimg\" -DPACKAGE_TARNAME=\"zimg\" -DPACKAGE_VERSION=\"2.1.0\" -DPACKAGE_STRING=\"zimg\ 2.1.0\" -DPACKAGE_BUGREPORT=\"https://github.com/sekrit-twc/zimg/pulls\" -DPACKAGE_URL=\"https://github.com/sekrit-twc/zimg\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DZIMG_X86=1 -I. -I/var/build/portage/media-libs/zimg-9999/work/zimg-9999    -std=gnu++11 -O2 -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -DNDEBUG -march=corei7-avx -O2 -pipe -c -o src/testcommon/win32_bitmap.o /var/build/portage/media-libs/zimg-9999/work/zimg-9999/src/testcommon/win32_bitmap.cpp
Assembler messages:
Fatal error: can't create src/testcommon/win32_bitmap.o: No such file or directory
/bin/sh ./libtool  --tag=CXX   --mode=compile x86_64-pc-linux-gnu-g++ -DPACKAGE_NAME=\"zimg\" -DPACKAGE_TARNAME=\"zimg\" -DPACKAGE_VERSION=\"2.1.0\" -DPACKAGE_STRING=\"zimg\ 2.1.0\" -DPACKAGE_BUGREPORT=\"https://github.com/sekrit-twc/zimg/pulls\" -DPACKAGE_URL=\"https://github.com/sekrit-twc/zimg\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DZIMG_X86=1 -I. -I/var/build/portage/media-libs/zimg-9999/work/zimg-9999  -I/var/build/portage/media-libs/zimg-9999/work/zimg-9999/src/zimg  -std=gnu++11 -O2 -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -DNDEBUG -march=corei7-avx -O2 -pipe -c -o src/zimg/api/libzimg_internal_la-zimg.lo `test -f 'src/zimg/api/zimg.cpp' || echo '/var/build/portage/media-libs/zimg-9999/work/zimg-9999/'`src/zimg/api/zimg.cpp
libtool: compile:  x86_64-pc-linux-gnu-g++ -DPACKAGE_NAME=\"zimg\" -DPACKAGE_TARNAME=\"zimg\" -DPACKAGE_VERSION=\"2.1.0\" "-DPACKAGE_STRING=\"zimg 2.1.0\"" -DPACKAGE_BUGREPORT=\"https://github.com/sekrit-twc/zimg/pulls\" -DPACKAGE_URL=\"https://github.com/sekrit-twc/zimg\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DZIMG_X86=1 -I. -I/var/build/portage/media-libs/zimg-9999/work/zimg-9999 -I/var/build/portage/media-libs/zimg-9999/work/zimg-9999/src/zimg -std=gnu++11 -O2 -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -DNDEBUG -mavx2 -mf16c -mfma -march=corei7-avx -O2 -pipe -c /var/build/portage/media-libs/zimg-9999/work/zimg-9999/src/zimg/resize/resize_impl_avx2.cpp  -fPIC -DPIC -o src/zimg/resize/.libs/libavx2_la-resize_impl_avx2.o
Makefile:1894: recipe for target 'src/testcommon/win32_bitmap.o' failed
make[1]: *** [src/testcommon/win32_bitmap.o] Error 2
make[1]: *** Waiting for unfinished jobs....
libtool: compile:  x86_64-pc-linux-gnu-g++ -DPACKAGE_NAME=\"zimg\" -DPACKAGE_TARNAME=\"zimg\" -DPACKAGE_VERSION=\"2.1.0\" "-DPACKAGE_STRING=\"zimg 2.1.0\"" -DPACKAGE_BUGREPORT=\"https://github.com/sekrit-twc/zimg/pulls\" -DPACKAGE_URL=\"https://github.com/sekrit-twc/zimg\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_PTHREAD_PRIO_INHERIT=1 -DHAVE_PTHREAD=1 -DZIMG_X86=1 -I. -I/var/build/portage/media-libs/zimg-9999/work/zimg-9999 -I/var/build/portage/media-libs/zimg-9999/work/zimg-9999/src/zimg -std=gnu++11 -O2 -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -DNDEBUG -march=corei7-avx -O2 -pipe -c /var/build/portage/media-libs/zimg-9999/work/zimg-9999/src/zimg/api/zimg.cpp  -fPIC -DPIC -o src/zimg/api/.libs/libzimg_internal_la-zimg.o
make[1]: Leaving directory '/var/build/portage/media-libs/zimg-9999/work/zimg-9999-abi_x86_64.amd64'
Makefile:2821: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
 * ERROR: media-libs/zimg-9999::vapoursynth failed (compile phase):
 *   emake failed
 *
 * If you need support, post the output of `emerge --info '=media-libs/zimg-9999::vapoursynth'`,
 * the complete build log and the output of `emerge -pqv '=media-libs/zimg-9999::vapoursynth'`.
 * The complete build log is located at '/var/build/portage/media-libs/zimg-9999/temp/build.log'.
 * The ebuild environment file is located at '/var/build/portage/media-libs/zimg-9999/temp/environment'.
 * Working directory: '/var/build/portage/media-libs/zimg-9999/work/zimg-9999-abi_x86_64.amd64'
 * S: '/var/build/portage/media-libs/zimg-9999/work/zimg-9999'
```

```
=================================================================
                        Package Settings
=================================================================

media-libs/zimg-9999::vapoursynth was built with the following:
USE="-cli -debug -static-libs" ABI_X86="64 -32 -x32" CPU_FLAGS_X86="sse"
```

**Comment by 4re** [comment]
Should be fixed in bf0fdf0ebcbd5c40996f9ef684bb3ab95e1f2757

**Comment by Kagami** [comment]
Now it works. Thanks.
